setup.py

```diff
@@ -21,6 +22,11 @@ def read_file(filename):
     'jsonschema',
 ]
 
+if sys.version_info < (3,):
```

Comment:
This seems to have introduced a problem with the wheel for Python 3 (reopened #303).